### ISSUE TITLE
Learning rate for shared weights

### ISSUE THREAD

**Issue by mariemabid** [keras-team/keras#8306]
In the paper ``Efficient BackProp`` by LeCun, it wa recommanded the following for initializing the learning rate:

> If shared weights are used such as in time-delay neural networks (TDNN) or convolutional neural networks, the learning rate should be proportional to the square root of number of connections sharing the weight, because we know the gradients are a sum of more or less independent terms

How can I implement this in Keras?

And how about learning rate per weight?
